src/main/java/com/example/application/backend/entities/AddressEntity.java

```diff
@@ -7,10 +7,10 @@
 /**
  * This is a basic address class.
  *
- * @author  Jessica Reistel, Laura Neuendorf and Sabrine Gamdou
- * @version 2.0
+ * @author  Jessica Reistel, Laura Neuendorf, Monika Martius and Sabrine Gamdou
+ * @version 3.0
  * @since   21-12-2020
- * @lastUpdated 05.01.2021
+ * @lastUpdated 25.01.2021
```

Comment:
Wenn etwas bearbeitet wurde kommt das in lastUpdated rein mit Datum und dann from PersonXY, Person ZY,... :)

Reply:
Das haben wir nie so gemacht xD

Reply:
Vor allem sind ja dann die namen wieder draußen, wenn jemand das neu updated..